augur/data/schema-multiplot-collections.json

```diff
@@ -0,0 +1,119 @@
+{
+    "$schema": "http://json-schema.org/draft-06/schema#",
+    "$id": "https://nextstrain.org/schemas/multiplot/v1",
+    "title": "Nextstrain multiplot data schema proposal with collection at the top level",
+    "type": "object",
+    "additionalProperties": false,
+    "required": ["collections"],
+    "properties": {
+        "default_collection": {
+            "description": "Default collection to display where the value must be the `key` in one of the objects of the collections array. Optional -- if not provided, first collection will be displayed",
+            "type": "string"
+        },
+        "collections": {
+            "description": "Collections of measurements and their configurations for display in auspice",
+            "type": "array",
+            "minItems": 1,
+            "items": {
+                "description": "A single collection of related measurements and the collection's display config",
+                "type": "object",
+                "additionalProperties": false,
+                "required": ["key", "config", "measurements"],
+                "properties": {
+                    "key": {
+                        "description": "The short name of the collection that is only used internally within auspice. Each collection is expected to have a unique key.",
+                        "type": "string"
+                    },
+                    "config": {
```

Comment:
I'm generally happy with where this schema is sitting now - good enough to develop the prototype for sure!

If we got rid of `config` and just put those properties onto the `collections` object (like you've done with `key` when moving from e098483 to d13acc7) would that be simpler?